doc/engineering-reference/src/climate-sky-and-solar-shading-calculations/climate-calculations.tex

```diff
@@ -44,86 +44,79 @@ \subsection{EnergyPlus Design Day Temperature Calculations}\label{energyplus-des
 
 \subsection{Sky Radiation Modeling}\label{sky-radiation-modeling}
 
-EnergyPlus calculates the Horizontal Infrared Radiation Intensity in Wh/m\(^{2,}\) if it is missing on the weather file or for design days, from the Opaque Sky Cover field as shown in the following explanation.
+EnergyPlus calculates the Horizontal Infrared Radiation ($IR_H$), if it is missing in the weather file or for design days, from the Dry Bulb Temperature, Dewpoint Temperature, and Opaque Cloud Cover as described below.
 
-\begin{equation}
-Horizontal\_IR = Sk{y_{emissivity}}\cdot Sigma\cdot Temperature_{drybulb}^4
-\end{equation}
-
-where
-
-Horizontal\_IR = horizontal IR intensity \{W/m\(^{2}\)\}
-
-Sky\(_{emissivity}\) = sky emissivity
-
-Sigma = Stefan-Boltzmann constant = 5.6697e-8 \{W/m\(^{2}\)-K\(^{4}\)\}
-
-Temperature\(_{drybulb}\) = drybulb temperature \{K\}
-
-By default, the clear sky emissivity is calculated by the Clark \& Allen (1978) model as
+$IR_H$ is defined as the rate of infrared radiation emitted from the sky falling on a horizontal upward-facing surface, in \si{\watt\per\m\squared}.
 
 \begin{equation}
-Sky_{clear-sky-emissivity} = 0.787 + 0.764\cdot \ln \frac {T_{dewpoint}}{T_{Kelvin}}
+IR_H = \sigma \cdot T_{sky}^4
 \end{equation}
 
-where, T\(_{dewpoint}\) = dewpoint temperature \{K\}.
+\noindent where:
+\newline
 
-Alternatively, the clear sky emissivity can also be calculated using different models set by the user from options using the \textbf{WeatherProperty:SkyTemperature} object, including the calibrated version of the Berdahl and Martin, Brunt and Idso models (2017), listed as follows.
```

Comment:
@mitchute New sky temp models were added in #7562. It appears that these changes are removing some of the new content?

Reply:
Ah, I did not see that. Let me review and update this.